skbio/io/format/tests/test_embed.py

```diff
@@ -25,6 +27,60 @@
     _protein_to_vector, _vector_to_protein
 )
 
+from transformers import T5Tokenizer, T5EncoderModel
+from tqdm import tqdm
+import torch
+import re
+
+class TestWriteError(TestCase):
+    def test_write_function(self):
+        model_name = "Rostlab/prot_t5_xl_uniref50"
+        tokenizer_name = "Rostlab/prot_t5_xl_uniref50"
+        sequence_list = ["MKTVRQERLKSIVRILERSKEPVSGAQLAEELSVSRQVIVQDIAYLRSLGYNIVATPRGYVLAGG", "MKTVRQERLKSIVRILERSKEPVSGAQLAEELSVSRQVIVQDIAYLRSLGYNIVATPRGYVLAGG"]
```

Comment:
```suggestion
        sequence_list = ["MKTVRQERLKSIVRILERSKEPVSGAQLAEELSVSRQVIVQDIAYLRSLGYNIVATPRGYVLAGG", 
                                     "MKTVRQERLKSIVRILERSKEPVSGAQLAEELSVSRQVIVQDIAYLRSLGYNIVATPRGYVLAGG"]
```
This will probably break styling